### DISCUSSION TITLE
Act log just shows "Error: timeout"

### DISCUSSION THREAD

**Discussion by JoshMcCullough** [nektos/act#1808] [Q&A]
That's the entirity of the log. Not sure what to do about that -- seems to be hit or miss, sometimes the same workflow will work fine. I'm using `act-js`.

And to be clear, the workflow runs quickly (e.g. under a minute), so it's not the timeout config of the workflow/action as far as I understand.

Verbose logoutput:

> time="2023-05-16T23:36:44-04:00" level=debug msg="Loading environment from /path/to/project/_act/my-test-suite/001/repo/test/.env"
> time="2023-05-16T23:36:44-04:00" level=debug msg="Conditional GET for notices etag=1617f43b-a877-4569-a6e7-27c349bb2e04"
> time="2023-05-16T23:36:44-04:00" level=debug msg="Loading action inputs from /path/to/project/_act/my-test-suite/001/repo/test/.input"
> time="2023-05-16T23:36:44-04:00" level=debug msg="Loading secrets from /path/to/project/_act/my-test-suite/001/repo/test/.secrets"
> time="2023-05-16T23:36:44-04:00" level=debug msg="Evaluated matrix inclusions: map[]"
> time="2023-05-16T23:36:44-04:00" level=debug msg="Loading workflow '/path/to/project/_act/my-test-suite/001/repo/test/test.yml'"
> time="2023-05-16T23:36:44-04:00" level=debug msg="Reading workflow '/path/to/project/_act/my-test-suite/001/repo/test/test.yml'"
> time="2023-05-16T23:36:44-04:00" level=debug msg="Using first passed in arguments event for filtering: push"
> time="2023-05-16T23:36:44-04:00" level=debug msg="Preparing plan for a event: push"
> time="2023-05-16T23:36:44-04:00" level=debug msg="Using first passed in arguments event: push"
> time="2023-05-16T23:36:44-04:00" level=debug msg="Planning jobs for event: push"
> time="2023-05-16T23:36:45-04:00" level=debug msg="invalid character 'I' looking for beginning of value"
> Error: timeout

**Comment by ZauberNerd** [comment]
Did you try running act with `-v` for verbose output?

**Reply by JoshMcCullough** [reply]
:man_facepalming: Sorry, I thought I was already in verbose mode. Will try that next time it crops up. Thanks.

**Reply by JoshMcCullough** [reply]
@ZauberNerd -- added verbose log output to the OP.
